### DISCUSSION TITLE
How to disable OTLP Metrics Exporter

### DISCUSSION THREAD

**Discussion by enp** [open-telemetry/opentelemetry-java#2940] [Q&A]
I need for jaeger traces only, but I see message:
```
Caught: io.opentelemetry.sdk.autoconfigure.ConfigurationException: OTLP Metrics Exporter enabled but opentelemetry-exporter-otlp-metrics not found on classpath. Make sure to add it as a dependency to enable this feature.
io.opentelemetry.sdk.autoconfigure.ConfigurationException: OTLP Metrics Exporter enabled but opentelemetry-exporter-otlp-metrics not found on classpath. Make sure to add it as a dependency to enable this feature.
```
Please advice me how to disable OTLP Metrics Exporter without adding opentelemetry-exporter-otlp-metrics to classpath

**Comment by jkwatson** [comment]
This is probably a bug, but until we get it fixed, the easiest way to disable this is to set the system property `-Dotel.metrics.exporter=none`, or the equivalent env var: `OTEL_METRICS_EXPORTER=none`.

I'll log a bug about it, though; this isn't a great user experience, I agree.

**Reply by jkwatson** [reply]
https://github.com/open-telemetry/opentelemetry-java/issues/2941
